sylvain-pierrot/seasonal-workers
# feat(authentification): add keycloak server using docker compose

## What does this PR do ?

This PR is focused on implementing a **Keycloak** server for the authentication of seasonal workers. The implementation has been done using `Docker Compose`.

### How should this be manually tested ?

- Step 1: Create a `keycloak.env` file at the root of the project with the required configuration. Make sure to include the necessary environment variables and settings specific to your Keycloak server setup.

- Step 2: Build and start the Docker Compose setup for Keycloak using the following command:

```bash
docker compose up -d
```
- Step 3: Once the Docker Compose setup is running, navigate to the Keycloak admin console in your web browser. You can usually access it at http://localhost:8080. Log in with the admin credentials specified in your keycloak.env file.
